assets/src/components/PostSelector/PostSelector.js

```diff
@@ -30,15 +30,19 @@ export const PostSelector = (attributes) => {
    */
   const act_parent = window.p4ge_vars.planet4_options.act_page || null;
   const args = { per_page: -1, orderby: 'title', post_status: 'publish' };
-  const act_args = {
-    post_parent: act_parent,
-    ...args,
-  };
   const posts = useSelect((select) => {
     if ('post' === postType) {
       return [].concat(
         select('core').getEntityRecords('postType', 'post', {'include': selected}) || [],
-        select('core').getEntityRecords('planet4/v1', 'all-published-posts', args) || [],
+        select('core').getEntityRecords('planet4/v1', 'published', {post_type: postType, ...args}) || [],
+      );
+    }
+
+    if ('post,page' === postType) {
+      return [].concat(
```

Comment:
Just in case, here is another approach using destructuring (ES6) and avoid using `concat` (ES5). 
```
return [
     ...select('core').getEntityRecords('postType', 'post', {'include': selected}) || [],
     ...select('core').getEntityRecords('postType', 'page', {'include': selected}) || [],
     ...select('core').getEntityRecords('planet4/v1', 'published', {post_type: postType, ...args}) || [],
]
```
OFC this is not needed to change!